File: app/src/main/java/com/itba/runningMate/db/RunDb.java
package com.itba.runningMate.db;

import androidx.room.Database;
import androidx.room.RoomDatabase;
import androidx.room.TypeConverters;

@Database(entities = {RunEntity.class}, version = 3)
@TypeConverters({RunConverters.class})
public abstract class RunDb extends RoomDatabase {

    public static final String NAME = "run_db";
    private static RunDb instance;

    public abstract RunDao RunDao();
}

File: app/src/main/java/com/itba/runningMate/db/RunDb.kt
package com.itba.runningMate.db

import androidx.room.Database
import androidx.room.RoomDatabase
import androidx.room.TypeConverters

@Database(entities = [RunEntity::class], version = 4)
@TypeConverters(RunConverters::class)
abstract class RunDb : RoomDatabase() {
    abstract fun RunDao(): RunDao

    companion object {
        const val NAME = "run_db"
    }
}